Source/Test/Rewriting/Passes/MSTestRewriter.cs

```diff
@@ -141,7 +141,7 @@ internal void RewriteTestMethod(MethodDefinition method, MethodDefinition testMe
             if (isAsyncMethod)
             {
                 var funcType = this.Module.ImportReference(typeof(Func<>));
-                actionType = ImportGenericTypeInstance(this.Module, funcType, asyncReturnType);
```

Comment:
Right, removing `this.Module` makes sense, as we capture that info already by the visitor :)